assets/src/edit-story/components/helpCenter/quickTip/index.js

```diff
@@ -23,24 +23,32 @@ import styled from 'styled-components';
  */
 import { themeHelpers, Text, THEME_CONSTANTS } from '../../../../design-system';
 import { NAVIGATION_HEIGHT } from '../navigator/constants';
+import { TranslateWithMarkup } from '../../../../i18n';
 import { GUTTER_WIDTH } from '../constants';
 import { Transitioner } from './transitioner';
 
 const Panel = styled.div`
   width: 100%;
   padding-bottom: ${NAVIGATION_HEIGHT}px;
-
-  ${Text} + ${Text} {
-    margin-top: 8px;
-  }
 `;
 
 const Overflow = styled.div`
   position: relative;
   width: 100%;
   max-height: 70vh;
-  overflow-y: scroll;
   padding: ${GUTTER_WIDTH}px;
+
+  strong {
+    font-weight: 700;
+  }
+
+  .screenreader {
```

Comment:
Worth extracting this?

Dashboard has `visuallyHiddenStyles.js` for that.

I'd even make it a styled component that can be used anywhere:

```js
const VisuallyHidden = styled.span`
  position: absolute;
  height: 1px;
  width: 1px;
  overflow: hidden;
  clip: rect(1px, 1px, 1px, 1px);
  white-space: nowrap;
`;
```

Reply:
yea good call. I'll add it to the design system